conn.go

```diff
@@ -46,18 +46,6 @@ func newConn(conf connConfig, muted bool) (*conn, error) {
 	if err != nil {
 		return c, err
 	}
-	// When using UDP do a quick check to see if something is listening on the
```

Comment:
This is a duplicate of the previous PR's commit which needs removing.

Reply:
That is correct. Will rectify.